src/docs/sphinx/quickstart_guide/config_build.rst

```diff
@@ -103,14 +129,13 @@ Tool Dependencies
 .. _Shroud: https://shroud.readthedocs.io/en/develop/
 .. _Sphinx: http://www.sphinx-doc.org/en/master/
 
-Each tool has a corresponding build system variable (with the suffix ``_EXECUTABLE``)
-to supply the tool's executable path. For example, ``sphinx`` has a corresponding build
-system variable ``SPHINX_EXECUTABLE``.
+Each tool has a corresponding build system variable (with the suffix 
+``_EXECUTABLE``) to supply a path to the tool executable. For example, 
+``sphinx`` has a corresponding build system variable ``SPHINX_EXECUTABLE``.
 
-.. note::
-  To get a full list of all dependencies of Axom's dependencies in an ``uberenv``
-  build of our TPLs, please go to the TPL root directory and
-  run the following spack command ``./spack/bin/spack spec axom``.
+.. note:: To get a full list of all Axom dependencies in an ``uberenv``
+          build of our TPLs, please go to the TPL root directory and
+          run the following Spack command ``./spack/bin/spack spec axom``.
```

Comment:
This note should tell how to print dependencies in a vcpkg build as well as in a spack build.  It should also be moved to the end of the next section, after the discussion of uberenv, spack, and vcpkg.  I'd put it just before the line that contains `.. _building-axom-label:`, about 50 source lines down.  At that point, uberenv, spack, and vcpkg will be fully introduced (with diagrams) and fresh in the user's mind.

Possible rephrase:
```suggestion
.. note:: To get a full list of all Axom dependencies in an ``uberenv``
          ``spack`` build of our TPLs, please go to the TPL root directory
          and run the following Spack command: ``./spack/bin/spack spec axom``.
          The analogous command for an ``uberenv`` ``vcpkg`` build is:
          ``.\vcpkg depend-info axom``.
```

Reply:
good suggestion. change made.